frontend/components/Buttons.vue

```diff
@@ -1,18 +1,29 @@
 <template>
   <div class="btncontainer">
-    <button class="resetfilters" @click="resetButtons">
-      <i class="far fa-eye"></i>
+    <BaseButton
+      class="resetfilters"
+      @click="resetButtons"
```

Comment:
Does it work without `v-bind="$attrs"`? Did you test it?

Reply:
Yes, I did it this way because icon tag  i needs attributes of class, so if I try to use v-bind="$attrs" I can't give base button a class, because the i tag will inherit it, so in our case the icons will be doubled.